packages/docusaurus-theme-bootstrap/src/theme/CodeBlock/index.js

```diff
@@ -21,9 +21,13 @@ export default ({children, className}) => {
       {({style, tokens, getLineProps, getTokenProps}) => (
         <pre className={className} style={{...style, padding: '20px'}}>
```

Comment:
here we have pre, but we should add code too I think

Reply:
Thank you @slorber, I will look into it and make necessary changes... I really appreciate your help and I just learnt something new...